src/Entity/User.php

```diff
@@ -71,7 +71,7 @@ class User
     #[ORM\Column(length: 255)]
     private ?string $password = null;
 
-    #[ORM\Column(length: 255)]
+    #[ORM\Column(length: 191, unique: true)]
```

Comment:
When migrating it causes an error "index key too long"
I had this problem when working on snowtricks, it's because of utf8 and utf8mbi something like that
putting 191 char max fix the problem
Info here : 
https://stackoverflow.com/a/42047853/17685040